File: library/bottomsheets/src/main/java/com/nasahapps/mdt/bottomsheets/BottomSheetItem.java
package com.nasahapps.mdt.bottomsheets;

import android.graphics.drawable.Drawable;

import com.nasahapps.mdt.TitleIconItem;

/**
 * Created by hhasan on 10/14/16.
 */

public class BottomSheetItem extends TitleIconItem {

    private int menuId;

    public BottomSheetItem(CharSequence text, Drawable icon) {
        super(text, icon);
    }

    BottomSheetItem(CharSequence text, Drawable icon, int menuId) {
        super(text, icon);
        this.menuId = menuId;
    }

    public int getMenuId() {
        return menuId;
    }
}

File: library/bottomsheets/src/main/java/com/nasahapps/mdt/bottomsheets/BottomSheetItem.kt
package com.nasahapps.mdt.bottomsheets

import android.graphics.drawable.Drawable

/**
 * Created by hhasan on 10/14/16.
 */

class BottomSheetItem {

    var text: CharSequence? = null
        private set
    var icon: Drawable? = null
        private set
    val menuId: Int

    @JvmOverloads
    constructor(text: CharSequence, icon: Drawable, menuId: Int = 0) {
        this.text = text
        this.icon = icon
        this.menuId = menuId
    }
}
